tests/PHPStan/Testing/TestCase.php

```diff
@@ -39,7 +39,7 @@
 	public function getContainer(): \Nette\DI\Container
 	{
 		if (self::$container === null) {
-			$rootDir = __DIR__ . '/../..';
+			$rootDir = __DIR__ . '/../../..';
```

Comment:
Please do not move files from `src/Testing` elsewhere, it's supposed to be under `src/` so that it can be used in 3rd party repos.

Reply:
Thing is, checker complains about `extends \PHPUnit\Framework\TestCase`. But I guess I could whitelist it in config. 👍

Reply:
Yes, please do. The current way would hamper the usability a lot.

Reply:
Already done :)